crates/driver/src/infra/solver/dto/notification.rs

```diff
@@ -64,12 +70,23 @@ pub struct Notification {
 #[serde(rename_all = "lowercase")]
 pub enum Kind {
     EmptySolution,
+    SimulationFailed(Tx),
     ScoringFailed(ScoreKind),
     NonBufferableTokensUsed(BTreeSet<H160>),
     SolverAccountInsufficientBalance(U256),
     DuplicatedSolutionId,
 }
 
+#[serde_as]
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Tx {
+    pub from: H160,
+    pub to: H160,
+    pub input: Vec<u8>,
```

Comment:
Should probably be adjusted to send a hex string instead of vector of numbers.

Reply:
true